config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -62,17 +62,9 @@ presubmits:
       preset-bazel-remote-cache-enabled: "true"
```

Comment:
why don't we just remove this instead of adding more hacks to the build-then-unit script?

Reply:
The changes to the script are temporary. By updating the script we get good data from the presubmit. 

Changing the job def does not have this property. 

Also, the change to the script is targeting the pull-test-infra-bazel job. So removing the preset from the pull-test-infra-bazel-rbe job isn't helpful. 

I'm making the pull-test-infra-bazel-rbe job the same as the pull-test-infra-bazel job so I can validate the simpler image works.

Reply:
We can remove it from both, it's simpler. All this preset does is set that environment variable, which we are effectively ignoring.

Reply:
> By updating the script we get good data from the presubmit.
> Changing the job def does not have this property.

Could you explain this? I don't see how that is true.

👍 For making the jobs match to check the simpler image.

Reply:
> Could you explain this? 

Changes to the job def aren't applied until after merge. 

Changes to the code the jobs runs inside the container (like this hack script) are tested by the presubmit. 

So this PR already tests that we pass all tests on RBE. 

That will no longer be true if I change the blocking job, which is why I don't want to do it.